frontend/src/metabase/notifications/NotificationsActionsMenu/DashboardSubscriptionMenuItem.tsx

```diff
@@ -26,19 +27,24 @@ export function DashboardSubscriptionMenuItem({
     (dashCard) => !["text", "heading"].includes(dashCard.card.display),
   );
 
+  const hasAnySubscriptionChannel = hasEmailSetup || hasSlackSetup;
+
   if (!canManageSubscriptions || !hasDataCards) {
     return null;
   }
 
-  if (!isAdmin && !hasEmailSetup) {
+  if (!isAdmin && !hasAnySubscriptionChannel) {
     return (
       <Menu.Item
         data-testid="dashboard-subscription-menu-item"
         leftSection={<Icon name="subscription" />}
         disabled
       >
         <Text size="md" c="inherit">{t`Can't send subscriptions`}</Text>
-        <Text size="sm" c="inherit">{t`Ask your admin to set up email`}</Text>
+        <Text
+          size="sm"
+          c="inherit"
+        >{t`Ask your admin to set up email, or Slack`}</Text>
```

Comment:
```suggestion
        >{t`Ask your admin to set up email or Slack`}</Text>
```

Reply:
The dialog from the admin POV also has a final comma - didn't like it too much either. Want me to drop that one as well?

<img width="450" height="374" alt="image" src="https://github.com/user-attachments/assets/57c9a99a-f2ba-4231-8fe8-6aaecd3c6e37" />

Reply:
this is a hotly debated issue that doesn't matter at all. Personally, I'm in favor of the [oxford comma](https://en.wikipedia.org/wiki/Serial_comma). You can go either way with lists of 3 or more items, but never use a comma with 2 items.